### ISSUE TITLE
ImportError: No module named pygtk_chart

### ISSUE THREAD

**Issue by rahns** [sioutisc/indicator-internode#3]
Indicator-internode suddenly stopped working for me, so I decided to try reinstallation. I uninstalled it using simple-uninstall.sh, then installed it with `sudo python setup.py install`. The install completed successfully, although when I start the program with `indicator-internode` it gets up to importing pygtk_chart then fails with `ImportError: No module named pygtk_chart`
I went to `/usr/local/lib/python2.7/dist-packages/indicator_internode-12.10.1-py2.7.egg/` and sure enough there was nothing named pygtk_chart in any of the subdirectories.

**Comment by rahns** [comment]
I've got it working. The problem is with the zip file listed in the README.md. When this package is used I get the error mentioned above, but when directly cloning the source on GitHub I get no such error. 
I have indicator-internode running as it was before the reinstall. 
Now when running indicator-internode I get: 
`File "/usr/lib/python2.7/urllib2.py", line 1198, in do_open`
    `raise URLError(err)`
`urllib2.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)>`
I've tried running `indicator-internode --no-check-certificate` but get the same error.

**Comment by sioutisc** [comment]
just added point release that includes patch for missing pygtk_chart error

**Comment by sioutisc** [comment]
can't reproduce the SSL error

**Comment by rahns** [comment]
How should I go about troubleshooting it? Or would it be better to just disable checking the cert?

**Comment by rahns** [comment]
`PYTHONHTTPSVERIFY=0 indicator-internode` worked as a bypass, though I was able to solve the issue. First I ran `strace indicator-internode` and found python was trying to read CA Certificates from `/usr/lib/ssl`, which didn't exist. My CA Certificates are stored in the default Ubuntu location, which is `/etc/ssl`, so to solve the problem I created a soft link from /etc/ssl to /usr/lib/ssl with `sudo ln -s /etc/ssl /usr/lib/ssl` then indicator-internode ran without any issues.
